ql/src/test/org/apache/hadoop/hive/ql/metadata/TestHiveMetaStoreClientApiArgumentsChecker.java

```diff
@@ -146,6 +151,26 @@ public void testGetPartitionsByNames3() throws HiveException {
     hive.getPartitionsByNames(t, new ArrayList<>(), true);
   }
 
+  @Test
+  public void testGetPartitionsByNamesWithSingleBatch() throws HiveException {
+    hive.getPartitionsByNames(t, Arrays.asList("Greece", "Italy"), true);
```

Comment:
I don't see any assertions - how these tests are working?

Reply:
The assertions are in `TestHiveMetaStoreClient`.